docs/examples/image_processing/index.rst

```diff
@@ -0,0 +1,15 @@
+Image Processing
+================
+
+.. toctree::
+   :maxdepth: 2
+
+   data_loading/index
```

Comment:
I don't think that data loading should go there. Yes, we use images as an examples, but it doesn't make this example relevant only to this particular use case.

Reply:
Done